libs/designsystem/src/lib/components/chart/configs/type.config.ts

```diff
@@ -98,4 +111,127 @@ export const CHART_TYPES_CONFIG: ChartTypesConfig = {
       },
     },
   },
+  stock: {
+    type: 'line',
+    options: {
+      // locale is overridden in handleLocalization().
+      locale: 'en-US',
+      responsive: true,
+      maintainAspectRatio: false,
+      animation: {
+        duration: 0,
+      },
+      layout: {
+        padding: {
+          left: 0,
+          right: 10,
+          top: 30,
+          bottom: 0,
+        },
+      },
+      backgroundColor: getThemeColorRgbString('semi-light', 0.5),
+      scales: {
+        x: {
+          type: 'time',
+          // Time is overridden in handleLocalization().
+          time: {},
+          grid: {
+            lineWidth: 0,
+          },
+          ticks: {
+            maxRotation: 0,
+            source: 'data',
+            autoSkipPadding: 40,
+            font: {
+              size: 11,
+            },
+          },
+        },
+        y: {
+          position: 'right',
+          display: true,
+          grid: {
+            drawBorder: false,
+          },
+          ticks: {
+            display: true,
+            font: {
+              size: parseInt(fontSize('xs')),
+            },
+          },
+        },
+      },
+      elements: {
+        line: {
+          tension: 0, // Smooth curve (0 = no smoothing)
+          borderWidth: 2,
+        },
+        point: {
+          hitRadius: 20,
+          radius: 0,
+          hoverRadius: 8,
+          hoverBackgroundColor: getThemeColorHexString('primary'),
+          hoverBorderWidth: 0,
+        },
+      },
+      normalized: true,
+      hover: {
+        axis: 'xy',
+        mode: 'nearest',
+        intersect: false,
+      },
+      plugins: {
+        tooltip: {
+          mode: 'nearest',
+          intersect: false,
+          caretSize: 1,
+          caretPadding: 12,
+          displayColors: false,
+          backgroundColor: getThemeColorHexString('semi-light'),
+          padding: 8,
+          titleColor: 'black',
+          bodyColor: 'black',
+          titleFont: {
+            weight: 'normal',
+            size: 12,
```

Comment:
Consider hooking it up to design tokens
```suggestion
            size: parseInt(fontSize('xs')),
```

Reply:
remains of drb-chart :)